wandb/wandb_config.py

```diff
@@ -172,21 +172,26 @@ def update(self, params, allow_val_change=False):
         if not isinstance(params, dict):
             # Handle some cases where params is not a dictionary
             # by trying to convert it into a dictionary
+            if "__name__" in dir(params) and params.__name__ in ('tensorflow.python.platform.flags', 'absl.flags'):
+                params = params.FLAGS
```

Comment:
I added this incase someone pass in `tf.app.flags` directly, might be silly but the error message is confusing when I hit it.